### ISSUE TITLE
Add picture/image to excel

### ISSUE THREAD

**Issue by ryanlin1986** [SheetJS/js-xlsx#364]
Is is possible? I can not find any document about this.

**Comment by asitpanda** [comment]
I am looking for this also, Can we get some work around ?

**Comment by grimace** [comment]
Yes, I am looking for this also.  Any way to drag and drop an image into a cell?

**Comment by jdstafford** [comment]
+1

**Comment by saluker** [comment]
Is anyone working on this?  Can my company "sponsor" this feature or otherwise contribute financially to get this feature prioritized?  Our requirement is simple: ability to embed images into generated xlsx files.  Alternatively, we could read() a XLSX file with an image, change some data, and output that.

**Comment by bleuscyther** [comment]
+1

**Comment by bleuscyther** [comment]
https://github.com/SheetJS/js-xlsx/issues/137

**Comment by peek4y** [comment]
No updates on this? Is this project dead?

**Comment by Arunvairavan** [comment]
is possible

**Comment by ismarf9** [comment]
@Arunvairavan the links is broked :(

**Comment by dmastag** [comment]
I think he wanted to link to another npm library
https://www.npmjs.com/package/exceljs#images

**Comment by XiaoSunXS** [comment]
Hi, does anyone know how to get the pro version? We'd love to use it for inserting images and styling, but it seems there's no response from SheetJS's side for over 10 days
